code/__DEFINES/objects.dm

```diff
@@ -157,4 +157,21 @@ var/list/RESTRICTED_CAMERA_NETWORKS = list( //Those networks can only be accesse
 #define SENTRY_ALERT_DAMAGE				4
 #define SENTRY_ALERT_BATTERY			5
 #define SENTRY_ALERT_DELAY				200 //20 seconds
-#define SENTRY_DAMAGE_ALERT_DELAY		50 //5 seconds
\ No newline at end of file
+#define SENTRY_DAMAGE_ALERT_DELAY		50 //5 seconds
+
+
+//Scout cloak defines
+#define SCOUT_CLOAK_ENERGY	100
+#define SCOUT_CLOAK_STEALTH_DELAY 30
+#define SCOUT_CLOAK_RUN_DRAIN	5
+#define SCOUT_CLOAK_WALK_DRAIN	1
+#define SCOUT_CLOAK_ACTIVE_RECOVERY -5 //You only get this once every obj tick, so it'll be comparable to the inactive value
+#define SCOUT_CLOAK_INACTIVE_RECOVERY 5
+#define SCOUT_CLOAK_COOLDOWN 100
+#define SCOUT_CLOAK_TIMER 50
+#define SCOUT_CLOAK_RUN_ALPHA 128
+#define SCOUT_CLOAK_WALK_ALPHA 51
+#define SCOUT_CLOAK_STILL_ALPHA 13
+#define SCOUT_CLOAK_MAX_ENERGY 100
+#define SCOUT_CLOAK_OFF_DAMAGE 1
```

Comment:
Separate these bitflags into a headed section and give them a different name to avoid confusion.

Reply:
And this?

Reply:
Isn't it already headed?

Also what is meant by a 'different name'? I think SCOUT_CLOAK is pretty clear.

Reply:
`SCOUT_CLOAK_OFF_DAMAGE` and `SCOUT_CLOAK_OFF_ATTACK` are bitflags whereas the others are actual values.   They should have a name that differentiates them as bitflags and be separated for the other things that arent bitflags